ubench.h

```diff
@@ -46,6 +46,22 @@
    TODO: add a UBENCH_NOINLINE onto the macro generated functions to fix this.
 */
 #pragma warning(disable : 4711)
+
+/*
+   Disable warning about replacing undefined preprocessor macro '__cplusplus' with
+   0 emitted from microsofts own headers.
+   See: https://developercommunity.visualstudio.com/t/issue-in-corecrth-header-results-in-an-undefined-m/433021
+*/
+#pragma warning(disable : 4668)
+
+/*
+    Disabled warning about dangerous use of section.
+    section '.CRT$XCU' is reserved for C++ dynamic initialization. Manually 
+    creating the section will interfere with C++ dynamic initialization and may lead to undefined behavior
+*/
+#pragma warning(disable : 5247)
```

Comment:
I had to disable these as well to get it through the autobuilder. I would GUESS this is a new warning from a newer msvc version?
However this might not be the solution you want but I felt that it was not part of this work to fix it.

Reply:
That's fine for now. I think you are right and it is a new warning.